src/main/java/org/itsallcode/openfasttrace/importer/legacytag/config/PathConfig.java

```diff
@@ -116,9 +98,8 @@ public String getCoveredItemNamePrefix()
     @Override
     public String toString()
     {
-        return "PathConfig [pattern=" + this.pattern + ", pathMatcher=" + this.pathMatcher
-                + ", coveredItemNamePrefix=" + this.coveredItemNamePrefix
-                + ", coveredItemArtifactType=" + this.coveredItemArtifactType + ", tagArtifactType="
-                + this.tagArtifactType + "]";
+        return "PathConfig [pathMatcher=" + this.pathMatcher.toString() + ", coveredItemNamePrefix="
```

Comment:
Maybe enclose the comma separated parts in quotes for better readability of the output?

Reply:
:heavy_check_mark: